strings/loot_data/tags/obj.txt

```diff
@@ -1991,6 +1991,7 @@
 /obj/item/weapon/gun_upgrade/mechanism/overshooter    blacklisted=0    spawn_value=5    price_tag=100   100    
 /obj/item/weapon/gun_upgrade/mechanism/glass_widow    blacklisted=1    spawn_value=2    price_tag=100   100    
 /obj/item/weapon/gun_upgrade/mechanism/weintraub    blacklisted=0    spawn_value=3.33333    price_tag=100   100    
+/obj/item/weapon/gun_upgrade/mechanism/funswitch    blacklisted=0    spawn_value=3.33333    price_tag=100   100    
```

Comment:
This is an automatically generated file and doesn't need to be changed for this PR.

Reply:
Should hopefully be removed from the PR.